test/e2e/helpers/parallel_test_helper.go

```diff
@@ -136,7 +139,7 @@ func ParallelTestHelper(t *testing.T,
 			// verify the logger service receives the correct transformed event
 			eventTracker.AssertExact(1, recordevents.MatchEvent(
 				HasSource(eventSource),
-				recordevents.DataContains(tc.expected),
+				DataContains(tc.expected),
```

Comment:
Not your doing, but I'm curious about the conformance test having an exactly once delivery semantics.